css-grid-1/grid-items/grid-item-containing-block-001.xht

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
I chose xhtml via democratic election of other files in directory ;)
I.ll change that.